src/components/fields/derived.js

```diff
@@ -588,6 +591,14 @@ export const HoverInfo = connectToContainer(UnconnectedFlaglist, {
         {label: _('Norm'), value: 'norm'},
         {label: _('Divergence'), value: 'divergence'},
       ];
+    } else if (container.type === 'sunburst') {
+      options = [];
+      if (container.labels) {
```

Comment:
can we align this with `pie` above plz? in pie we don't do the container checks but should we?